src/test/java/io/codeka/gaia/e2e/SeleniumIT.java

```diff
@@ -31,6 +31,7 @@
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @Tag("e2e")
+@Disabled("Disabled until complete migration of client")
 public class SeleniumIT {
```

Comment:
You can also disable these `e2e` tests in the `.travis-ci.yml` configuration.
It will cause percy to be disabled in our PR too.

Reply:
:heavy_check_mark: